packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -396,6 +397,88 @@ abstract class ProjectBase extends Resource implements IProject {
   }
 }
 
+/**
+ * Statuses for Log Configurations
+ */
+export enum LogStatus {
+  /**
+   * Enables the log configuration
+   */
+  ENABLED='ENABLED',
+
+  /**
+   * Disables the log configuration
+   */
+  DISABLED='DISABLED'
+}
+
+/**
+ * Information about logs built to an S3 bucket for a build project.
+ */
+export interface S3LogsConfig {
+  /**
+   * Encrypt the S3 build log output
+   * @default true
+   */
+  readonly encrypted?: boolean;
+
+  /**
+   * The S3 Bucket to send logs to
+   */
+  readonly bucket: s3.IBucket;
+
+  /**
+   * The path prefix for S3 logs
+   * @default - no prefix
+   */
+  readonly prefix?: string;
+
+  /**
+   * The current status of the logs in Amazon CloudWatch Logs for a build project
+   * @default ENABLED
+   */
+  readonly status?: LogStatus;
+}
+
+/**
+ * Information about logs built to an S3 bucket for a build project.
+ */
+export interface CloudwatchLogsConfig {
+  /**
+   * The S3 Bucket to send logs to
+   */
+  readonly logGroup: logs.ILogGroup;
+
+  /**
+   * The prefix of the stream name of the Amazon CloudWatch Logs
+   * @default - no prefix
+   */
+  readonly prefix?: string;
+
+  /**
+   * The current status of the logs in Amazon CloudWatch Logs for a build project
+   * @default ENABLED
+   */
+  readonly status?: LogStatus;
```

Comment:
I don't see the point of this property, or this enum (why would I create a disabled log configuration...?). Let's just remove this for now, if there's demand for this feature, we'll add it back in (probably as a `enabled?: boolean` property).

Reply:
Fair point!